bothub/health/checks.py

```diff
@@ -1,23 +1,38 @@
+import logging
+
+from rest_framework import status
+
+
+logger = logging.getLogger('bothub.health.checks')
+
+CHECK_ACCESSIBLE_API_URL = '/api/repositories/'
```

Comment:
Use the python-decouple to get this value from envvar, and let the /api/repositories as default value.